src/Components/Components/src/ParameterView.cs

```diff
@@ -88,8 +88,7 @@ public bool TryGetValue<TValue>(string parameterName, [MaybeNullWhen(false)] out
         /// <typeparam name="TValue">The type of the value.</typeparam>
         /// <param name="parameterName">The name of the parameter.</param>
         /// <returns>The parameter value if found; otherwise the default value for the specified type.</returns>
-        [return: MaybeNull]
-        public TValue GetValueOrDefault<TValue>(string parameterName)
+        public TValue? GetValueOrDefault<TValue>(string parameterName)
             => GetValueOrDefault<TValue>(parameterName, default!);
```

Comment:
```suggestion
            => GetValueOrDefault<TValue>(parameterName, default);
```

Reply:
The API this calls does not allow nulls: https://github.com/dotnet/aspnetcore/pull/25261/files/aa545c6bf61275f3ccaad13fb8c66b49d6b37658#diff-31999040ba5519a0e4866136e0f2e568R102.

Reply:
So can we not do this?

```suggestion
            => GetValueOrDefault<TValue?>(parameterName, default);
```